### DISCUSSION TITLE
DB-GPT: SQL-GEN and SQL-diagnosis and based knowledge qa

### DISCUSSION THREAD

**Discussion by csunny** [oceanbase/oceanbase#1382] [Ideas]
i build a DB-GPT project，which is here.  https://github.com/csunny/DB-GPT ,  and it's really cool. 

There is some example.
![image](https://user-images.githubusercontent.com/17919400/236997157-d4e4e744-2db6-43c0-a6ea-ff7407a79bd4.png)

![image](https://user-images.githubusercontent.com/17919400/236997215-c526a4d4-86f1-4ae4-a3a1-aac066319430.png)

**Comment by hnwyllmm** [comment]
看起来很有意思。我看到可以用它部署数据库，可以部署OceanBase吗？

**Reply by csunny** [reply]
可以的
